cargo-dist/templates/ci/github/partials/publish_homebrew.yml.j2

```diff
@@ -24,6 +24,16 @@
           pattern: artifacts-*
           path: Formula/
           merge-multiple: true
+      # Set up and cache homebrew for formatting formulas
+      - name: Set up Homebrew
```

Comment:
We shouldn't need to do this. [Homebrew is preinstalled](https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2004-Readme.md#package-management) on the Linux runners we use as the global runners; we don't need to reinstall it. You should just be able to perform `brew update` before `brew style --fix` to make sure it's the latest version.

Reply:
Had no idea, thank you! Fixed.